### ISSUE TITLE
"Cannot read property 'slice' of undefined"

### ISSUE THREAD

**Issue by eglaysher** [urbit/bridge#240]
Went to `littel-ponnys.azimuth.network` to try using bridge to do a personal breach so I could get ponnys back on the network. Went to the Networking tab. Clicked the "Trigger New Continuity Era" check box. Clicked the Reset Transaction button. Got "Cannot read property 'slice' of undefined" underneath.

Here's all the console output:
```
2useKeyfileGenerator.js:65 seed is nondeterminable for revision 1
AdminNetworkingKeys.js:85 deriving seed with revision 2
useKeyfileGenerator.js:76 keys do not match details for revision 1
useEthereumTransaction.js:59 TypeError: Cannot read property 'slice' of undefined
    at trezor.js:26
    at u (runtime.js:45)
    at Generator._invoke (runtime.js:271)
    at Generator.e.<computed> [as next] (runtime.js:97)
    at r (asyncToGenerator.js:3)
    at o (asyncToGenerator.js:25)
```

**Comment by Fang-** [comment]
Hmm. Maybe we're behind on Trezor lib again? I would expect it to complain about that before failing like this. Will investigate, thanks!

**Comment by eglaysher** [comment]
Also of note is that when I initially set ~littel-ponnys networking keys, I had to manually set a key since that wasn't yet supported with a trezor. There seems to be an error related to that in the output above?

**Comment by Fang-** [comment]
That's more of a debug printf, which should maybe be stripped out. We deal with known vs unknown cases differently (ie doesn't give you your keyfile if we can't derive known-good keys).

**Comment by shrugs** [comment]
closing since this should be solved with #244
